api/static/rest_framework_swagger/css/osf_screen.css

```diff
@@ -0,0 +1,1232 @@
+/* http://meyerweb.com/eric/tools/css/reset/ v2.0 | 20110126 */
+.swagger-ui-wrap html,
```

Comment:
Why is that @sloria? Since these are additions to rest_framework_swagger, don't we want to version them?